mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -214,7 +217,7 @@ def connect(self, port, addr='localhost', uid=UNKNOWN_UID,
                     logging.exception("Failed to create socket connection!")
                     raise
                 time.sleep(1)
-
+        self.port = port
```

Comment:
I expect in 100% of cases that addr will be localhost. Actually I was considering removing it. People will be specifying just 'port' so the ordering shouldn't matter much.